### PR TITLE
Add carrier logos.

### LINKED ISSUE #2029: UPS logo for UPS rate responses
After https://github.com/Automattic/woocommerce-connect-server/issues/1370 is finished, enabled sites will receive UPS carrier rates. We'll need to display the rates with the UPS logo in the rates list:

image

https://wcship.wordpress.com/2020/05/15/adding-a-carrier-account-in-wcs-i4-design/

### PR DESCRIPTION
**Pull request by c-shultz**
Fixes #2029 

Adds UPS and USPS logos to rates list.

To test: generate rates for USPS and UPS and make sure logos appear OK.

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -1,5 +1,17 @@
...
+* Add   - Carrier logos next to rates.
+
+= 1.23.2 - 2020-06-12 =
+* Fix   - Refund not possible on order page.
+
+= 1.23.1 - 2020-06-10 =
+* Tweak - Update WooCommerce compatibility to 4.2
+* Fix	- Taxjar broken in admin's new order page
+* Fix   - Tax recalculation on admin order screen missing street address.
+
```
Comment (c-shultz) [inline changelog.txt:14]: I needed to add an entry for this change but the changelog was out-of-date so I copied in all the entries from develop.

**Comment by harriswong** [comment]
👀

**Comment by harriswong** [comment]
@c-shultz changelog has a conflict with latest `develop`, can you try merging latest develop please?

**Comment by c-shultz** [comment]
I beat you to it! 😄

**Comment by harriswong** [comment]
👍 🚢

**Pull request merged**